src/Symfony/Component/RateLimiter/Policy/TokenBucketLimiter.php

```diff
@@ -9,11 +9,16 @@
  * file that was distributed with this source code.
  */
 
-namespace Symfony\Component\RateLimiter;
+namespace Symfony\Component\RateLimiter\Policy;
 
 use Symfony\Component\Lock\LockInterface;
 use Symfony\Component\Lock\NoLock;
 use Symfony\Component\RateLimiter\Exception\MaxWaitDurationExceededException;
+use Symfony\Component\RateLimiter\LimiterInterface;
+use Symfony\Component\RateLimiter\Rate;
```

Comment:
I think `Rate` should maybe also be moved in the new `Policy` namespace? It's only used when creating a `TokenBucketLimiter`.

Reply:
I agree.